packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1517,7 +1515,7 @@ int _compareData<T>(
   T b,
   ColumnData column,
   SortDirection direction, {
-  ColumnData secondarySortColumn,
+  ColumnData? secondarySortColumn,
```

Comment:
add generic type T to both ColumnData types in this parameter list

Reply:
done